### PR TITLE
Feat: add host sync from Tencent and CRUD

### PR DESCRIPTION
**Pull request by lbzss**
Feat: add host sync from Tencent and CRUD

### CODE REVIEW

File: api/helloworld/v1/greeter.proto [hunk 1]
```diff
@@ -45,6 +45,39 @@ service Greeter {
...
+    option (google.api.http) = {
+      delete: "/cmdb/deleteInstance/{InstanceId}",
+    };
+  };
+  rpc UpdateInstance(UpdateInstanceRequest) returns (UpdateInstanceReply){
+    option (google.api.http) = {
+      put: "/cmdb/updateInstance/{InstanceId}",
+      body: "*",
+    };
+  };
```
Comment (1ch0) [inline api/helloworld/v1/greeter.proto:74]: 1. Make the definitions more standardized and RESTful
2. Use lowercase camelCase for method and parameter names

improved code example：
```proto
  rpc GetInstance(GetInstanceRequest) returns (GetInstanceReply) {
    option (google.api.http) = {
      get: "/cmdb/instance/{instanceId}",
    };
  };
```

File: ent/schema/cloudhost.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+type CloudHost struct {
+	ent.Schema
+}
+
+// Fields of the CloudHost.
+func (CloudHost) Fields() []ent.Field {
+	return []ent.Field{
+		field.String("instanceId").NotEmpty().Unique(),
+		field.String("vpcId").NotEmpty(),
+		field.String("SubnetId"),
```
Comment (1ch0) [inline ent/schema/cloudhost.go:18]: Use lowercase camelCase

File: api/helloworld/v1/greeter.proto [hunk 3]
```diff
@@ -128,6 +161,74 @@ message HelloReply {
...
+  string InstanceName = 4;
+  string InstanceState = 5;
+  int64 CPU = 6;
+  int64 Memory = 7;
+  string CreatedTime = 8;
+  string InstanceType = 9;
+  int64 EniLimit = 10;
+  int64 EnilpLimit = 11;
+  int64 InstanceEniCount = 12;
+}
```
Comment (1ch0) [inline api/helloworld/v1/greeter.proto:177]: Use lowercase camelCase

File: api/helloworld/v1/greeter.proto [hunk 4]
```diff
@@ -128,6 +161,74 @@ message HelloReply {
...
+  Host Instance = 1;
+}
+
+message CreateInstanceRequest {
+  Host Instance = 1;
+}
+
+message CreateInstanceReply {
+  Host Instance = 1;
+}
```
Comment (1ch0) [inline api/helloworld/v1/greeter.proto:205]: Use lowercase camelCase

File: api/helloworld/v1/greeter.proto [hunk 5]
```diff
@@ -128,6 +161,74 @@ message HelloReply {
...
+  string Message = 2;
+}
+
+message DeleteInstanceRequest {
+  string InstanceId = 1;
+}
+
+message DeleteInstanceReply {
+  string Message = 1;
+}
```
Comment (1ch0) [inline api/helloworld/v1/greeter.proto:230]: Use lowercase camelCase

File: internal/data/host.go [hunk 6]
```diff
@@ -0,0 +1,181 @@
+package data
+
+import (
+	"context"
+	"fmt"
+	"github.com/go-kratos/kratos/v2/log"
+	"kubecit/ent/cloudhost"
+	"kubecit/internal/biz"
+	"reflect"
+)
```
Comment (1ch0) [inline internal/data/host.go:10]: Imports should be grouped as standard library, third party packages, own packages.

improved code example：
```go
import (
	"context"
	"fmt"
	"reflect"

	"github.com/go-kratos/kratos/v2/log"
	
	"kubecit/ent/cloudhost"
	"kubecit/internal/biz"
)
```

File: internal/data/host.go [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+		dField, _ := dt.FieldByName(fieldName)
+		dFieldValue := dv.FieldByName(fieldName)
+		if sField.Name == dField.Name && dFieldValue.CanSet() && sField.Type == dField.Type {
+			dFieldValue.Set(sFieldValue)
+		} else if sField.Name == dField.Name && sFieldValue.CanConvert(dFieldValue.Type()) {
+			dFieldValue.Set(sFieldValue.Convert(dFieldValue.Type()))
+		}
+	}
+	return nil
+}
```
Comment (1ch0) [inline internal/data/host.go:181]: 👍

**Review by 1ch0 — COMMENTED** [review]
👍
Just a few minor improvements needed regarding code conventions

File: api/helloworld/v1/greeter.proto (hunk 1 above)
Comment (lbzss) [reply api/helloworld/v1/greeter.proto]: Those methods name and parameters name have been modified. Please check again

File: internal/data/host.go (hunk 6 above)
Comment (lbzss) [reply internal/data/host.go]: Modified

File: api/helloworld/v1/greeter.proto [hunk 8]
```diff
@@ -47,34 +47,34 @@ service Greeter {
...
     option (google.api.http) = {
-      post: "/cmdb/createInstance",
+      post: "/cmdb/instance",
       body: "*",
     };
   };
   rpc ListInstances(ListInstancesRequest) returns (ListInstancesReply){
     option (google.api.http) = {
-      get: "/cmdb/listInstances",
+      get: "/cmdb/instance",
```
Comment (1ch0) [inline api/helloworld/v1/greeter.proto:61]: improved code example：
```proto
get: "/cmdb/instances"
```

File: api/helloworld/v1/greeter.proto [hunk 9]
```diff
@@ -47,34 +47,34 @@ service Greeter {
...
     option (google.api.http) = {
-      put: "/cmdb/updateInstance/{InstanceId}",
+      put: "/cmdb/instance/{instanceId}",
       body: "*",
     };
   };
   rpc SyncFromTencent(SyncFromTencentRequest) returns (SyncFromTencentReply) {
     option (google.api.http) = {
-      post: "/cmdb/SyncFromTencent",
+      post: "/cmdb/sync-from-tencent",
```
Comment (1ch0) [inline api/helloworld/v1/greeter.proto:77]: improved code example：
```proto
get: "/cmdb/sync/tencent"
```

**Pull request merged**